File: core/Entities/Admin/DLForwardCategoryOfficial.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace core.Entities.Admin
{
    public class DLForwardCategoryOfficial: BaseEntity
    {
        public DLForwardCategoryOfficial()
		{
		}

        public DLForwardCategoryOfficial(int customerofficialid, DateTime dateforwarded, string emailid, string phoneno, string whatsappno, int loggedinempid)
        {
            CustomerOfficialId = customerofficialid;
            DateTimeForwarded = dateforwarded;
            DateOnlyForwarded = dateforwarded.Date;
            EmailIdForwardedTo = emailid;
            PhoneNoForwardedTo = phoneno;
            WhatsAppNoForwardedTo = whatsappno;
            LoggedInEmployeeId = loggedinempid;
        }

		public int DLForwardCategoryId { get; set; }
        public int OrderItemId {get; set;}
        public int CustomerOfficialId { get; set; }
        public string AgentName {get; set;}
        public DateTime DateTimeForwarded { get; set; }=DateTime.Now;
        public DateTime DateOnlyForwarded {get; set;}=DateTime.Now.Date;
        public string EmailIdForwardedTo { get; set; }
        public string PhoneNoForwardedTo { get; set; }
        public string WhatsAppNoForwardedTo { get; set; }
        public int LoggedInEmployeeId {get; set;}

    }
}

File: client2/src/app/shared/models/admin/dlForwardCategoryOfficial.ts
export interface IDLForwardCategoryOfficial
{
	checked: boolean;
	id: number;
	dLForwardItemId: number;
	orderItemId: number;
	customerOfficialId: number;
	agentName: string;
	dateTimeForwarded: Date;
	dateOnlyForwarded: Date;
	emailIdForwardedTo: string;
	phoneNoForwardedTo: string;
	whatsAppNoForwardedTo: String;
	loggedInEmployeeId: number;
}

export class dLForwardCategoryOfficial implements IDLForwardCategoryOfficial
{
	checked = false;
	id= 0;
	dLForwardItemId= 0;
	orderItemId= 0;
	customerOfficialId= 0;
	agentName= '';
	dateTimeForwarded= new Date('1900-01-01');
	dateOnlyForwarded= new Date('1900-01-01');
	emailIdForwardedTo= '';
	phoneNoForwardedTo= '';
	whatsAppNoForwardedTo= '';
	loggedInEmployeeId= 0;
}